src/main/java/study/step4/RacingGame.java

```diff
@@ -0,0 +1,46 @@
+package study.step4;
+
+import study.step4.domain.Car;
+import study.step4.move.MoveStrategy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingGame {
+
+    private final List<Car> carList = new ArrayList<>();
```

Comment:
```suggestion
    private final List<Car> cars = new ArrayList<>();
```
자료구조에 의존하지 않는 변수명을 사용하는 것을 권장하고 있습니다.
그래서 보통 컬렉션을 담고있는 변수명은 주로 복수형으로 표현하고 있어요
사실 거의 변경이 없겠지만 자료구조가 변경된다는 이유로 변수명이 함께 변경되는 것은 좋지 않은 것 같아요